bundles/org.openhab.binding.renault/README.md

```diff
@@ -30,6 +30,7 @@ You require your MyRenault credential, locale and VIN for your MyRenault registe
 | vin               | Vehicle Identification Number.                                             | yes      |
 | refreshInterval   | Interval the car is polled in minutes.                                     | no       |
 | updateDelay       | How long to wait for commands to reach car and update to server in seconds.| no       |
+| kamereonApiKey    | Kamereon API Key.                                                          | no       | 
```

Comment:
This parameter is marked as required in the XML description.

Reply:
This is "use the default" more that "required".  Like the refreshInterval and updateDelay.  I will set the default value in the RenaultConfiguration and remove the required flag in the XML.  It means there are 2 sources of truth of the defaults in the code for this but the user has an easier time.